blocks/organizational-cards/organizational-cards.css

```diff
@@ -323,6 +314,20 @@ main .section > div.organizational-cards-wrapper {
     right: 0;
   }
 
+  .organizational-card.collapsed .card-content-wrapper {
+    padding-bottom: unset;
+    border-bottom: unset;
+  }
+
+  .organizational-card .card-content-wrapper .collapse-btn {
+    background: var(--primary-blue-50);
+    border: 1px solid var(--primary-blue-50);
```

Comment:
all variables that starts with `primary` are by us and we will get rid of them soon. I am approving this PR but kindly follow up with appropriate new vars in new PR.